cmd/minikube/cmd/version.go

```diff
@@ -43,13 +48,57 @@ var versionCmd = &cobra.Command{
 			"minikubeVersion": minikubeVersion,
 			"commit":          gitCommitID,
 		}
+
+		if listPackagesVersions {
+			co := mustload.Running(ClusterFlagValue())
+			runner := co.CP.Runner
+			// docker version --format='{{.Client.Version}}'
+			// sudo crictl version
+			// runc --version
+			// crio version
+			// buildctl --version
+			// sudo ctr version
+			// sudo podman version
+
+			versionCMDS := map[string]*exec.Cmd{
+				"docker":     exec.Command("docker", "version", "--format={{.Client.Version}}"),
```

Comment:
any reason why for docker we filter out everything but client version,  but for other tools we return everything we got, including commits, build versions, build dates, etc?

Reply:
Most likely we want the server version, not the client version (which is just the "web browser" here).

Reply:
Would be nice to have both, actually. When  docker container runtime is used, we use docker cli a lot, and it would be useful to see here both client and server versions to be able to catch version mismatches due to an failed upgrade, or image misconfiguration.

Reply:
the reason for using the client version is, I want this to work on all runtimes (and since we install this ourseleves the client and server are same version)

when then service is not running server version will give errors